test/jdk/java/net/Inet4Address/PingThis.java

```diff
@@ -27,9 +27,14 @@
 
 /* @test
  * @bug 7163874 8133015
+ * @requires os.family != "windows" && os.family != "aix"
  * @library /test/lib
  * @summary InetAddress.isReachable is returning false
```

Comment:
'&&' double ampersand syntax does not work here. Should be a single ampersand as suggested by Aleksei.

Reply:
Once a C programmer, hard to resist. Thx. :)